public/app/app.states.js

```diff
@@ -81,7 +81,7 @@
         'templateUrl': '/app/dashboard/profile.html',
         'controller': 'ProfileController'
       })
-      .state('dashboard.profile.user', {
+      .state('userprofile', {
```

Comment:
Oddly, `$state.go(...)` in profileController was unhappy with dots in state name (above). So, I changed the state name. Duh!